entrypoint.sh

```diff
@@ -8,9 +8,9 @@ groupadd --non-unique --gid ${DOCKER_GROUP} dockeronhost
 usermod -aG dockeronhost node
 
 # compatibility: initial volume setup
-chown node:node /app/cache
-chown node:node /app/compiles
-chown node:node /app/db
+mkdir -p /app/cache && chown node:node /app/cache
+mkdir -p /app/compiles && chown node:node /app/compiles
+mkdir -p /app/db && chown node:node /app/db
```

Comment:
Do we need to create `/app/output` here as well? cc @briangough

Reply:
Looks like we do, yes.  I had missed those.

Reply:
Added `/app/output` now